File: src/main/java/me/skynda/vehicle/entities/VehicleReview.java
package me.skynda.vehicle.entities;

import javax.persistence.*;
import javax.validation.constraints.NotNull;

import lombok.Data;
import me.skynda.image.entities.Image;

import java.io.Serializable;

@Entity
@Data
@Table(name = "vehicle_review")
public class VehicleReview implements Serializable {
	
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer id;

	@ManyToOne(fetch = FetchType.LAZY)
	@JoinColumn(name = "logo_id", nullable = false)
	@NotNull
	private Image logo;

	@ManyToOne(fetch = FetchType.LAZY)
	@JoinColumn(name = "video_id", nullable = false)
	@NotNull
	private Image video;

	@Column(name = "text")
	private String text;

	@Column(name = "rating")
	private Integer rating;

	@Column(name = "vehicle_id", nullable = false)
	private Integer vehicleId;

}

File: skynda_dotnet_backend/Triven.Data.EntityFramework/Entities/VehicleReview.cs
using System.ComponentModel.DataAnnotations.Schema;
using Triven.Data.EntityFramework.Entities.Base;
using Triven.Domain.Models;

namespace Triven.Data.EntityFramework.Entities
{
    [Table("VehicleReview")]
    public class VehicleReview: AuditableModel, IVehicleReview
    {
        public int LogoId { get; set; }
        public int VideoId { get; set; }

        public string VideoUrl { get; set; }

        public string LogoUrl { get; set; }

        public int Rating { get; set; }

        public string Text { get; set; }

        public int VehicleId { get; set; }

        public virtual Vehicle Vehicle { get; set; }
    }
}